### PR TITLE
Improve tooling

### PR DESCRIPTION
**Pull request by hect95**
- Install Autoware related msgs
- Add `X11` forwarding support for visualising docker UI in host machine
- Install Plotjuggler
- Change Cyclone DDS min socket receive buffer size to 15 MB to
  avoid issues with big messages
- Add `front_camera_top_lidar.txt` topic list file for recording
- Add `check_rosbag.py` script to analyse MCAP rosbags and examinate data loss
- Set Docker `runtime` stage`CMD` to record rosbag with `sensor_topics.txt` list by default
- Add `-p` flag in `runtime.sh` to define host rosbags directory and `-l` to define Cyclone DDS xml cfg file

### CODE REVIEW

File: cyclone_dds.xml
Comment (GreatAlexander) [inline cyclone_dds.xml:1]: We should add the car interface to this file to match the rest of repos

File: dev.sh
Comment (GreatAlexander) [inline dev.sh:1]: We should update the dev and runtime scripts to add the improvements we made in the other repos

File: cyclone_dds.xml
Comment (hect95) [reply cyclone_dds.xml:1]: Done

File: dev.sh
Comment (hect95) [reply dev.sh:1]: Done

**Review by GreatAlexander — APPROVED** [review]

**Pull request merged**